StaticAnalyzer/models.py

```diff
@@ -10,136 +10,92 @@ class RecentScansDB(models.Model):
 
 
 class StaticAnalyzerAndroid(models.Model):
-    TITLE = models.CharField(max_length=50)
-    APP_NAME = models.CharField(max_length=260)
-    REAL_NAME = models.CharField(max_length=255)
+    FILE_NAME = models.CharField(max_length=260)
+    APP_NAME = models.CharField(max_length=255)
+    APP_TYPE = models.TextField()
```

Comment:
TextFiled()?
Not equals with `StaticAnalyzerIPA`
`APP_TYPE = models.CharField(max_length=20, default='')`

Reply:
good idea.